### PR TITLE
Fixes linting issues

### PR DESCRIPTION
**Pull request by peduarte**
Preparation for an alpha release

### CODE REVIEW

File: packages/css/src/index.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ const processStyleObject = (
 ) => {
   // key: css prop or override or a selector
   // value is: cssValue, a util, specificity prop, or
-  for (const key in obj) {
+  for (const key of Object.keys(obj)) {
```
Comment (hadihallak) [inline packages/css/src/index.ts:90]: is this a linting thing? looks very weird.

**Review by hadihallak — APPROVED** [review]
LGTM. let's create a task to fix things that are typed as `any` throughout the codebase

File: packages/css/src/index.ts (hunk 1 above)
Comment (hadihallak) [reply packages/css/src/index.ts:90]: https://github.com/modulz/stitches/issues/118

**Comment by peduarte** [comment]
Def

**Pull request merged**